deps.ml

```diff
@@ -0,0 +1,144 @@
+open Types
+open Dune_items
+
+type t = Dir_set.t Libraries.t
+
+let dune_describe_external_lib_deps = Bos.Cmd.(v "dune" % "describe" % "external-lib-deps")
+
+let dune_describe_entries = Bos.Cmd.(v "dune" % "describe" % "package-entries")
```

Comment:
I think that these should not be top-level expressions. They are evaluated when the module is loaded, and if they fail, then the user cannot call simple things such as `opam-dune-lint --help`. They should be wrapped in `unit -> x` functions, or made lazy.
```suggestion
let dune_describe_external_lib_deps () = Bos.Cmd.(v "dune" % "describe" % "external-lib-deps")

let dune_describe_entries () = Bos.Cmd.(v "dune" % "describe" % "package-entries")
```